metadata_service/config.py

```diff
@@ -118,37 +114,30 @@ class LocalConfig(LocalGremlinConfig, Config):
     }
 
 
-# The databuilder expects this to be False currently. We are defaulting to true because the testing expects this
-if bool(distutils.util.strtobool(os.environ.get('IGNORE_NEPTUNE_SHARD', 'False'))):
-    shard_set_explicitly('')
+class GremlinConfig(LocalConfig, LocalGremlinConfig):
+    JANUS_GRAPH_URL = None
+
+
+class NeptuneConfig(LocalGremlinConfig, LocalConfig):
+    import boto3
 
+    def __init__(self):
+        from amundsen_gremlin.test_and_development_shard import shard_set_explicitly
+        # The databuilder expects this to be False currently. We are defaulting to true because the testing expects this
+        if bool(distutils.util.strtobool(os.environ.get('IGNORE_NEPTUNE_SHARD', 'False'))):
+            shard_set_explicitly('')
 
-class NeptuneConfig(LocalGremlinConfig, Config):
     DEBUG = False
-    TESTING = False
     LOG_LEVEL = 'INFO'
-    LOCAL_HOST = '0.0.0.0'
 
     # FORMAT: wss://<NEPTUNE_URL>:<NEPTUNE_PORT>/gremlin
     PROXY_HOST = os.environ.get('PROXY_HOST')
     PROXY_PORT = None
     PROXY_CLIENT = PROXY_CLIENTS['NEPTUNE']
```

Comment:
This class is inherited from Config, and these values should be set via the environment variable available instead of redeclaring them for different proxies. It will be easy to manage different proxies this way if we change any variable/config